src/main/java/com/faforever/client/chat/PlayerInfoBean.java

```diff
@@ -24,10 +29,17 @@
   private final StringProperty avatarTooltip;
   private final BooleanProperty friend;
   private final BooleanProperty foe;
+  //TODO I think this does not describe what it is meant for
```

Comment:
proposals? `channelsThisUserIsModeratorIn` is the next best thing I came up with